caption/doc/index.md

```diff
@@ -0,0 +1,43 @@
+```js script
+import '@divriots/dockit-core/caption/dockit-caption.define';
+import '@divriots/dockit-core/image-container/dockit-image-container.define';
+```
+
+# dockit-caption
+
+Component used to call out information in documentation pages.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/caption/dockit-caption.define';
```

Comment:
please follow the style of the project: all setups have an extension, because in general you can't assume people use a compiler, especially in the WC world